### DISCUSSION TITLE
Pro not available after renewing Student Developer Pack

### DISCUSSION THREAD

**Discussion by pexeixv** [community/community#45687] [GitHub Education] [answered]
I renewed my Student Developer Pack by submitting my college ID card. The application got approved, but I'm still being asked to pay $4/mo for GitHub Pro. How do I fix this issue?

![Screenshot_20230129-105135~2](https://user-images.githubusercontent.com/68476117/215306754-20db8188-52e8-49c7-bfc5-eddc5494d7a0.png)

**Comment by jge162** [comment, marked as answer]
Click on the link that says contact support in the email. And you should be able to submit a ticket to have it resolved. Or go here to [support link](https://support.github.com/request/education?tags=hubberfy_education)

**Reply by pexeixv** [reply]
Thanks
